lib/feature/auth/widgets/app_elevated_button.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:split/res/app_colors.dart';
+
+class AppButton extends StatelessWidget {
+  const AppButton({super.key, required this.title, this.onPressed});
+  final String title;
+  final void Function()? onPressed;
```

Comment:
pls explain why you use void Function instead of VoidCallBack and why it's nullable?!?!?!?